ext/pcre/config0.m4

```diff
@@ -130,5 +130,14 @@ else
     [no],,
     [$PHP_PCRE_CFLAGS])
   PHP_ADD_BUILD_DIR([$ext_builddir/pcre2lib])
-  PHP_INSTALL_HEADERS([ext/pcre], [php_pcre.h pcre2lib/])
+  PHP_INSTALL_HEADERS([ext/pcre], m4_normalize([php_pcre.h]))
+  PHP_INSTALL_HEADERS([ext/pcre/pcre2lib], m4_normalize([
+    config.h
+    pcre2.h
+    pcre2_internal.h
+    pcre2_intmodedep.h
+    pcre2_jit_neon_inc.h
+    pcre2_jit_simd_inc.h
+    pcre2_ucp.h
```

Comment:
```suggestion
    pcre2.h
```

The upstream library, for example, installs here only two headers - pcre2.h and pcre2posix.h (which isn't implemented in this bundled sources). Probably, other headers are also not needed in PHP installation here.

Reply:
Yeah. I'm also unsure about exporting all libgd headers. but we probably should leave that as is for now.